services/compute/galleryimage/client.go

```diff
@@ -39,10 +40,30 @@ func (c *GalleryImageClient) Get(ctx context.Context, location, name string) (*[
 
 // CreateOrUpdate methods invokes create or update on the client
 func (c *GalleryImageClient) CreateOrUpdate(ctx context.Context, location, imagePath, name string, compute *compute.GalleryImage) (*compute.GalleryImage, error) {
+	compute.SourceType = "local"
 	return c.internal.CreateOrUpdate(ctx, location, imagePath, name, compute)
 }
 
 // Delete methods invokes delete of the compute resource
 func (c *GalleryImageClient) Delete(ctx context.Context, location, name string) error {
 	return c.internal.Delete(ctx, location, name)
 }
+
+// UploadImageFromLocal   methods invokes  UploadImageFromLocal  on the client
+func (c *GalleryImageClient) UploadImageFromLocal(ctx context.Context, location, imagePath, name string, compute *compute.GalleryImage) (*compute.GalleryImage, error) {
+	compute.SourceType = "local"
```

Comment:
can you define "local", "sfs" as enums in moc and reuse them. It is not dev friendly to hard code these here.

Reply:
yeah I had that in the node agent code.  Brought it moc-sdk as well.